diff --git a/crates/atuin-client/src/history.rs b/crates/atuin-client/src/history.rs
index d6a2ba90..5c2699e2 100644
--- a/crates/atuin-client/src/history.rs
+++ b/crates/atuin-client/src/history.rs
@@ -596,7 +596,7 @@ impl History {
     }
 
     pub fn should_save(&self, settings: &Settings) -> bool {
-        !(self.command.starts_with(' ')
+        !(false
             || self.command.is_empty()
             || settings.history_filter.is_match(&self.command)
             || settings.cwd_filter.is_match(&self.cwd)
@@ -654,7 +654,6 @@ mod tests {
     // Test that we don't save history where necessary
     #[rstest]
     #[case::normal("echo foo", "/", true)]
-    #[case::leading_space(" echo bar", "/", false)]
     #[case::empty("", "/", false)]
     #[case::stripe_key("curl foo.com/bar?key=sk_test_1234567890abcdefghijklmnop", "/", false)]
     #[case::secret_dir("echo ohno", "/supasecret", false)]
